Update contributors.json

jhertz-apple opened the pull request:
Add myself to contributors.json<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/498f06773e035b705388566c52ab1d8b5fbb542b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/11446 "Failed to checkout and rebase branch from PR 15395") | [❌ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/11650 "Failed to checkout and rebase branch from PR 15395") | [❌ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/12004 "Failed to checkout and rebase branch from PR 15395") | [❌ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/13091 "Failed to checkout and rebase branch from PR 15395") | [❌ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/10921 "Failed to checkout and rebase branch from PR 15395") 
| [❌ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/11467 "Failed to checkout and rebase branch from PR 15395") | [❌ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/14035 "Failed to checkout and rebase branch from PR 15395") | [❌ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/11632 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/5/builds/13091 "Failed to checkout and rebase branch from PR 15395") | 
| [❌ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/11609 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/23/builds/14035 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 api-mac](https://ews-build.webkit.org/#/builders/14/builds/12004 "Failed to checkout and rebase branch from PR 15395") | [❌ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/13512 "Failed to checkout and rebase branch from PR 15395") | 
| [❌ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/11547 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/23/builds/14035 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/14/builds/12004 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/2/builds/13512 "Failed to checkout and rebase branch from PR 15395") | 
| [❌ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/9736 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 api-ios](https://ews-build.webkit.org/#/builders/23/builds/14035 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/14/builds/12004 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/2/builds/13512 "Failed to checkout and rebase branch from PR 15395") | 
| [❌ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/10866 "Failed to checkout and rebase branch from PR 15395") | [❌ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/10945 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/16/builds/11632 "Failed to checkout and rebase branch from PR 15395") | [❌ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/11537 "Failed to checkout and rebase branch from PR 15395") | 
| | [❌ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/10120 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/14/builds/12004 "Failed to checkout and rebase branch from PR 15395") | [❌ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/35/builds/11537 "Failed to checkout and rebase branch from PR 15395") | 
| | [❌ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/14393 "Failed to checkout and rebase branch from PR 15395") | | [❌ 🛠 jsc-mips](https://ews-build.webkit.org/#/builders/24/builds/11865 "Failed to checkout and rebase branch from PR 15395") | 
| | [❌ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/10801 "Failed to checkout and rebase branch from PR 15395") | | [❌ 🧪 jsc-mips-tests](https://ews-build.webkit.org/#/builders/24/builds/11865 "Failed to checkout and rebase branch from PR 15395") | 
<!--EWS-Status-Bubble-End-->

jhertz-apple committed:
Update contributors.json

Add myself to contributors.json

Commit:
[JSC] Align duplicate declaration checks in EvalDeclarationInstantiation with the spec
https://bugs.webkit.org/show_bug.cgi?id=167837
<rdar://problem/111328974>

Reviewed by Yusuke Suzuki.

For the sloppy-mode eval(), this change:

1. Removes slowish TypeError-throwing logic from executeEval() that also wasn't spec-compliant
   (SyntaxError should be raised instead), harmonizing error messages.

2. Expands resolveScopeForHoistingFuncDeclInEval() to be called for all declared variables, which
   currently includes function declarations as well, ensuring SyntaxError is thrown for duplicates
   with upper yet non-top lexical scopes [1], all while skipping CatchScopeWithSimpleParameter [2].

3. Introduces emitPutToScopeDynamic(), which circumvents default ResolveType resolution that isn't
   correct wrt skipping CatchScopeWithSimpleParameter as resolveScopeForHoistingFuncDeclInEval() does.

   We can't possibly tweak BytecodeGenerator::resolveType() to account for eval().

   This fixes both top-level and block-level function declarations to be hoisted correctly from eval()
   within simple parameter catch block by the same name.

4. Removes isExtensible() check from resolveScopeForHoistingFuncDeclInEval() because for declared
   variables, CanDeclareGlobalVar [3] is already implemented, while for Annex B hoisted functions,
   the implementation doesn't appear correct to unconditionally rely on isExtensible() even if the
   property is already present.

   Furthermore, performing CanDeclareGlobalVar in resolveScopeForHoistingFuncDeclInEval() is kinda
   superfluous given we put jsUndefined() variables in executeEval(), and results in incorrect
   error being thrown (SyntaxError instead of TypeError) if global object is non-extensible.

[1]: https://tc39.es/ecma262/#sec-evaldeclarationinstantiation (step 3.d.i.2.a.i)
[2]: https://tc39.es/ecma262/#sec-variablestatements-in-catch-blocks
[3]: https://tc39.es/ecma262/#sec-candeclareglobalvar

All JSTests changes were proven to align JSC with V8 and SpiderMonkey.

* JSTests/ChakraCore/test/Closures/bug_OS_2299723.baseline-jsc:
* JSTests/stress/const-not-strict-mode.js:
* JSTests/stress/eval-func-decl-in-eval-within-catch-scope.js:
* JSTests/stress/eval-func-decl-in-global-of-eval.js:
* JSTests/stress/eval-func-decl-within-eval-with-reassign-to-var.js:
* JSTests/stress/eval-let-const-redeclararion.js: Added.
* JSTests/stress/global-lexical-var-injection.js:
* JSTests/stress/lexical-let-not-strict-mode.js:
* JSTests/test262/expectations.yaml: Mark 160 tests as passing.
* Source/JavaScriptCore/bytecompiler/BytecodeGenerator.cpp:
(JSC::BytecodeGenerator::generate):
(JSC::BytecodeGenerator::hoistSloppyModeFunctionIfNecessary):
(JSC::BytecodeGenerator::emitPutToScopeDynamic):
* Source/JavaScriptCore/bytecompiler/BytecodeGenerator.h:
* Source/JavaScriptCore/interpreter/Interpreter.cpp:
(JSC::Interpreter::executeEval):
* Source/JavaScriptCore/runtime/JSScope.cpp:
(JSC::JSScope::resolveScopeForHoistingFuncDeclInEval):

Co-authored-by: Oleksandr Skachkov <gskachkov@gmail.com>
Canonical link: https://commits.webkit.org/265614@main

squelart committed:
Attachment: Use image element to show icon/thumbnail
https://bugs.webkit.org/show_bug.cgi?id=258437
rdar://105252742

Reviewed by Tim Nguyen.

Instead of using an inner attachment with legacy rendering, use a standard img element with a data URL
pointing at a blob that contains the image data.

Widespread changes:
- Removed m_innerLegacyAttachment and all related code.
- Removed image-only code.
- Consistently using "wide-layout" instead of "modern", and "narrow" instead of "legacy".

* LayoutTests/fast/attachment/mac/wide-attachment-image-controls-basic-expected.txt:
* LayoutTests/platform/ios-wk2/fast/attachment/cocoa/wide-attachment-rendering-expected.txt:
* LayoutTests/platform/mac-wk2/fast/attachment/cocoa/wide-attachment-rendering-expected.txt:
* Source/WebCore/editing/Editor.cpp:
(WebCore::Editor::didInsertAttachmentElement):
* Source/WebCore/html/HTMLAttachmentElement.cpp:
(WebCore::HTMLAttachmentElement::create):
(WebCore::HTMLAttachmentElement::didAddUserAgentShadowRoot):
(WebCore::attachmentPlaceholderIdentifier):
(WebCore::attachmentIconIdentifier):
(WebCore::HTMLAttachmentElement::ensureWideLayoutShadowTree):
(WebCore::HTMLAttachmentElement::updateProgress):
(WebCore::HTMLAttachmentElement::setFile):
(WebCore::HTMLAttachmentElement::attributeChanged):
(WebCore::HTMLAttachmentElement::attachmentActionForDisplay const):
(WebCore::HTMLAttachmentElement::attachmentTitleForDisplay const):
(WebCore::HTMLAttachmentElement::attachmentSubtitleForDisplay const):
(WebCore::HTMLAttachmentElement::updateAttributes):

(WebCore::HTMLAttachmentElement::updateImage):
This sets the img element with the correct "src" data contents (thumbnail or icon or nothing).

(WebCore::HTMLAttachmentElement::updateThumbnailForNarrowLayout):
(WebCore::HTMLAttachmentElement::updateThumbnailForWideLayout):
(WebCore::HTMLAttachmentElement::updateIconForNarrowLayout):
(WebCore::HTMLAttachmentElement::updateIconForWideLayout):
(WebCore::HTMLAttachmentElement::setNeedsWideLayoutIconRequest):
(WebCore::HTMLAttachmentElement::requestWideLayoutIconIfNeeded):
(WebCore::HTMLAttachmentElement::requestIconWithSize const):
(WebCore::attachmentPreviewIdentifier): Deleted.
(WebCore::HTMLAttachmentElement::ensureModernShadowTree): Deleted.
(WebCore::HTMLAttachmentElement::updateThumbnail): Deleted.
(WebCore::HTMLAttachmentElement::updateIcon): Deleted.
* Source/WebCore/html/HTMLAttachmentElement.h:
* Source/WebCore/html/shadow/attachmentElementShadow.css:
(div#attachment-preview-area):
(img#attachment-icon):
(attachment#attachment-preview): Deleted.
* Source/WebCore/rendering/AttachmentLayout.mm:
(WebCore::AttachmentLayout::AttachmentLayout):
* Source/WebCore/rendering/RenderAttachment.cpp:

(WebCore::RenderAttachment::paintWideLayoutAttachmentOnly const):
Call requestWideLayoutIconIfNeeded when painting the shadow tree.

* Source/WebCore/rendering/RenderThemeIOS.mm:
(WebCore::RenderThemeIOS::attachmentIntrinsicSize const):
(WebCore::RenderThemeIOS::paintAttachment):
* Source/WebCore/rendering/RenderThemeMac.mm:
(WebCore::RenderThemeMac::paintAttachment):

* Source/WebKit/WebProcess/WebPage/WebPage.cpp:
(WebKit::WebPage::updateAttachmentThumbnail):
(WebKit::WebPage::updateAttachmentIcon):
Provide data blobs to wide-layout attachments.

Canonical link: https://commits.webkit.org/265615@main

youennf committed:
Resync WPT webcodecs tests up to 4c14ced
https://bugs.webkit.org/show_bug.cgi?id=258614
rdar://111440403

Reviewed by Tim Nguyen.

* LayoutTests/imported/w3c/web-platform-tests/webcodecs/audio-encoder-codec-specific.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/audio-encoder-codec-specific.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/audio-encoder-codec-specific.https.any.js: Added.
(make_silent_audio_data):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/video-encoder-config.https.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/video-encoder-config.https.any.js:
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/video-encoder-config.https.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/video-encoder.https.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/video-encoder.https.any.js:
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/video-encoder.https.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/videoDecoder-codec-specific.https.any.js:
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/videoFrame-construction.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/videoFrame-construction.any.js:
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/videoFrame-serialization.crossAgentCluster.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webcodecs/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/ios-wk2/imported/w3c/web-platform-tests/webcodecs/videoFrame-construction.any-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/webcodecs/videoFrame-construction.any-expected.txt.

Canonical link: https://commits.webkit.org/265616@main

rwlbuis committed:
[contain-intrinsic-size] Support "auto none"
https://bugs.webkit.org/show_bug.cgi?id=258618

Reviewed by Tim Nguyen.

See:
https://github.com/w3c/csswg-drafts/pull/8989

* LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/auto-014-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/auto-014.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/parsing/contain-intrinsic-size-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/parsing/contain-intrinsic-size-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/parsing/contain-intrinsic-size-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/parsing/contain-intrinsic-size-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/parsing/contain-intrinsic-size-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/parsing/contain-intrinsic-size-valid.html:
* Source/WebCore/css/CSSProperties.json:
* Source/WebCore/css/ComputedStyleExtractor.cpp:
(WebCore::valueForContainIntrinsicSize):
* Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp:
(WebCore::CSSPropertyParserHelpers::consumeContainIntrinsicSize):
* Source/WebCore/dom/Document.cpp:
(WebCore::CallbackForContainIntrinsicSize):
* Source/WebCore/rendering/RenderBox.cpp:
(WebCore::RenderBox::explicitIntrinsicInnerWidth const):
(WebCore::RenderBox::explicitIntrinsicInnerHeight const):
* Source/WebCore/rendering/style/RenderStyle.h:
* Source/WebCore/rendering/style/RenderStyleConstants.cpp:
(WebCore::operator<<):
* Source/WebCore/rendering/style/RenderStyleConstants.h:
* Source/WebCore/rendering/style/RenderStyleInlines.h:
(WebCore::RenderStyle::containIntrinsicHeightHasAuto const):
(WebCore::RenderStyle::containIntrinsicWidthHasAuto const):
(WebCore::RenderStyle::hasAutoLengthContainIntrinsicSize const):
* Source/WebCore/rendering/updating/RenderTreeUpdater.cpp:
(WebCore::RenderTreeUpdater::updateElementRenderer):
* Source/WebCore/style/StyleBuilderCustom.h:
(WebCore::Style::BuilderCustom::applyValueContainIntrinsicWidth):
(WebCore::Style::BuilderCustom::applyValueContainIntrinsicHeight):

Canonical link: https://commits.webkit.org/265617@main

anttijk committed:
Nullptr crash in Layout::InlineItemsBuilder::collectInlineItems
https://bugs.webkit.org/show_bug.cgi?id=258664
rdar://111272076

Reviewed by Alan Baradlay.

Looks like a null item in LayoutQueue.

* Source/WTF/wtf/CheckedRef.h:
(WTF::downcast):

Add CheckedRef<const Foo> version of downcast<>.

* Source/WebCore/layout/formattingContexts/inline/InlineItemsBuilder.cpp:
(WebCore::Layout::traverseUntilDamaged):

Make LayoutQueue use CheckedRef.

(WebCore::Layout::initializeLayoutQueue):

Test for null formattingContextRoot.firstChild() first so we don't add a nullptr to LayoutQueue.

(WebCore::Layout::InlineItemsBuilder::collectInlineItems):

Canonical link: https://commits.webkit.org/265618@main

karlrackler committed:
Revert [265610@main] Regression: User-friendly name for the GPUProcess no longer gets set
https://bugs.webkit.org/show_bug.cgi?id=258600
Unreviewed revert
This reverts because it broke the build for the bots.

* Source/WebKit/GPUProcess/GPUProcess.cpp:
(WebKit::GPUProcess::initializeGPUProcess):
* Source/WebKit/GPUProcess/GPUProcess.h:
* Source/WebKit/GPUProcess/GPUProcessCreationParameters.cpp:
(WebKit::GPUProcessCreationParameters::encode const):
(WebKit::GPUProcessCreationParameters::decode):
* Source/WebKit/GPUProcess/GPUProcessCreationParameters.h:
* Source/WebKit/GPUProcess/cocoa/GPUProcessCocoa.mm:
(WebKit::GPUProcess::platformInitializeGPUProcess): Deleted.
* Source/WebKit/GPUProcess/mac/GPUProcessMac.mm:
(WebKit::GPUProcess::initializeProcessName):
(WebKit::GPUProcess::updateProcessName): Deleted.
* Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in:
* Source/WebKit/UIProcess/Cocoa/GPUProcessProxyCocoa.mm:
(WebKit::GPUProcessProxy::platformInitializeGPUProcessParameters):

Canonical link: https://commits.webkit.org/265619@main

JonWBedard commented on metadata/contributors.json:
```diff
@@ -3772,6 +3772,16 @@
+         "jesse_hertz@apple.com",
+         "jesse.hertz@apple.com"
+      ],
+      "nicks" : [
```
We should have a GitHub username, `jhertz-apple`, right?

jhertz-apple replied:
fixed and updated

obyknovenius committed:
[WPE] Implement `PlatformWebView` key press and button clicks simulation methods
https://bugs.webkit.org/show_bug.cgi?id=258665

Reviewed by Miguel Gomez.

In fact, this is a copy of the Playstation port implementation with
some minor changes.

We are extending the `WKPage` API with a couple of functions that can
handle keyboard and mouse events.

These API events are then converted to `NativeWebKeyboardEvent` and
`NativeWebMouseEvent` respectively and sent to `WebPageProxy' for handling.

* Source/WebKit/Shared/API/c/wpe/WKEventWPE.cpp: Added.
(WKKeyboardEventMake):
(WKMouseEventMake):
(WKWheelEventMake):
* Source/WebKit/Shared/API/c/wpe/WKEventWPE.h: Added.
* Source/WebKit/SourcesWPE.txt:
* Source/WebKit/UIProcess/API/C/wpe/WKPagePrivateWPE.cpp: Added.
(WKPageHandleKeyboardEvent):
(WKPageHandleMouseEvent):
* Source/WebKit/UIProcess/API/C/wpe/WKPagePrivateWPE.h: Added.
* Tools/TestWebKitAPI/PlatformWebView.h:
* Tools/TestWebKitAPI/glib/TestExpectations.json:
* Tools/TestWebKitAPI/wpe/PlatformWebViewWPE.cpp:
(TestWebKitAPI::PlatformWebView::simulateSpacebarKeyPress):
(TestWebKitAPI::PlatformWebView::simulateAltKeyPress):
(TestWebKitAPI::PlatformWebView::simulateRightClick):
(TestWebKitAPI::PlatformWebView::simulateButtonClick):
(TestWebKitAPI::PlatformWebView::simulateMouseMove):

Canonical link: https://commits.webkit.org/265620@main

sammygill committed:
Silence Window resize events while backgrounding Zillow on iOS
https://bugs.webkit.org/show_bug.cgi?id=258648
rdar://109480544

Reviewed by Brent Fulgham.

There is currently a bug on Zillow that will cause the map to reset
to its initial location if the window is sized to a small size (enough
to make the map disappear) and resized back to its original size. This is
caused when window resize events are triggered and cause unusual
behavior on iPad due to the background snapshotting behavior of Safari.

Since this process does not happen to other browsers, because they
do not participate in this snapshotting process, it results in a bad
experience to those using Safari. To bring the behavior that occurs in
Safari in line with other browsers we can add a quirk that disables
window resize events from triggering while backgrounding. This should
be removed when the bug is fixed on Zillow's end.

We will also start logging this behavior to console whenever it gets
triggered.

* Source/WebCore/page/LocalFrameView.cpp:
(WebCore::LocalFrameView::scheduleResizeEventIfNeeded):
* Source/WebCore/page/Quirks.cpp:
(WebCore::Quirks::shouldSilenceWindowResizeEvents const):

Canonical link: https://commits.webkit.org/265621@main

cdumez committed:
Regression: User-friendly name for the GPUProcess no longer gets set
https://bugs.webkit.org/show_bug.cgi?id=258600

Reviewed by Per Arne Vollan and Brent Fulgham.

Access to launch services has been blocked by the GPUProcess' sandbox. As a
result, the GPUProcess no longer gets a user-friendly name in Activity Monitor.

To address the issue, we now:
1. Have the UIProcess create a sandbox extension for launch services and send it
   to the GPUProcess to temporarily consume it during initialization
2. The GPUProcess, on initialization now consumes this extension
3. The GPUProcess checks in with Launch Services and sets the process name
4. The GPUProcess closes connections to launch services
5. The GPUProcess revokes the sandbox extension

* Source/WebKit/GPUProcess/GPUProcess.cpp:
(WebKit::GPUProcess::initializeGPUProcess):
* Source/WebKit/GPUProcess/GPUProcess.h:
* Source/WebKit/GPUProcess/GPUProcessCreationParameters.cpp:
(WebKit::GPUProcessCreationParameters::encode const):
(WebKit::GPUProcessCreationParameters::decode):
* Source/WebKit/GPUProcess/GPUProcessCreationParameters.h:
* Source/WebKit/GPUProcess/cocoa/GPUProcessCocoa.mm:
(WebKit::GPUProcess::platformInitializeGPUProcess):
* Source/WebKit/GPUProcess/mac/GPUProcessMac.mm:
(WebKit::GPUProcess::initializeProcessName):
(WebKit::GPUProcess::updateProcessName):
* Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in:
* Source/WebKit/UIProcess/Cocoa/GPUProcessProxyCocoa.mm:
(WebKit::GPUProcessProxy::platformInitializeGPUProcessParameters):

Canonical link: https://commits.webkit.org/265622@main

Commit:
Update contributors.json to include jhertz-apple

Reviewed by NOBODY (OOPS!).

* metadata/contributors.json:

cdumez reviewed (changes requested):
This patch seems to contain a lot of unrelated changes.
